helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -235,7 +239,22 @@ private Response batchGetStoppableInstances(String clusterId, JsonNode node, boo
             OBJECT_MAPPER.getTypeFactory().constructCollectionType(List.class, String.class));
         if (!orderOfZone.isEmpty() && random) {
           String message =
-              "Both 'orderOfZone' and 'random' parameters are set. Please specify only one option.";
+              "Both 'zone_order' and 'random' parameters are set. Please specify only one option.";
+          _logger.error(message);
+          return badRequest(message);
+        }
+      }
+
+      if (node.get(InstancesAccessor.InstancesProperties.to_be_stopped_instances.name()) != null) {
```

Comment:
Are we assuming this will be something passed by user? If it is already reconstructed by our code, then we don't need this check.

Reply:
Yes. This will be provided by user. We won't construct and save the `to_be_stopped_instances` in our code.